scripts/animatediff.py

```diff
@@ -217,6 +217,16 @@ def on_ui_settings():
             section=section
         )
     )
+    shared.opts.add_option(
+        "animatediff_default_save_formats",
+        shared.OptionInfo(
+            ["GIF", "PNG"],
+            "Default Save Formats",
+            gr.CheckboxGroup,
+            {"choices": supportedSaveFormats},
+            section=section
+        )
+    )
```

Comment:
This might be better near the top of the settings, right after "Path to save AnimateDiff motion modules". Then the output specific settings would be right below it, instead of above.